### PR TITLE
Cmake support

### LINKED ISSUE #21: Add CMake support

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #21

### CODE REVIEW

File: cmake/FindGSL.cmake [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Try to find gnu scientific library GSL
```
Comment (grzanka) [inline cmake/FindGSL.cmake]: Why not using default script https://cmake.org/cmake/help/latest/module/FindGSL.html ?

Comment (grzanka) [reply cmake/FindGSL.cmake]: It looks it is present even in cmake 3.2: https://cmake.org/cmake/help/v3.2/module/FindGSL.html

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 2.8)
```
Comment (grzanka) [inline CMakeLists.txt]: Version 2.8 dates back to 2009. Do we need to support such old versions ?
https://cmake.org/pipermail/cmake/2009-November/033303.html

Consider using 3.2 (with GSL support), released in 2015: https://cmake.org/pipermail/cmake/2015-March/060082.html

**Review by mateuszbaran — COMMENTED** [review]
Thanks, building `amtrack_demo`, `amtrack_plots` and `amtrack_test` works for me and I can debug them now. I think installation instructions need to be updated though (files `README_DEVELOPERS` and `INSTALL`).

**Review by mateuszbaran — APPROVED** [review]

**Pull request merged**